src/compiler/generate.ml

```diff
@@ -19,6 +20,67 @@ let check_auxiliary_output com actx =
 			Genjson.generate com.types file
 	end
 
+let export_hxb com root m =
+	match m.m_extra.m_kind with
+		| MCode | MMacro | MFake -> begin
+			(* Printf.eprintf "Export module %s\n" (s_type_path m.m_path); *)
+			let anon_identification = new Genshared.tanon_identification ([],"") in
+			let writer = new HxbWriter.hxb_writer com anon_identification in
+			writer#write_module m;
+			let ch = IO.output_bytes() in
+			let bytes_module = IO.close_out ch in
```

Comment:
I just came across this, what is it about? It looks like we're creating a channel and immediately close it...

Reply:
No idea :sweat_smile: